### PR TITLE
[liquidcheck] Initial contribution

### PR DESCRIPTION
**Pull request by marcelGoerentz**
<!--
Thanks for contributing to the openHAB project!
Please describe the goal and effect of your PR here.
Pay attention to the below notes and to *the guidelines* for this repository.
Feel free to delete any comment sections in the template (starting with "<!--").

ATTENTION: Don't use "git merge" when working with your pull request branch!
This can clutter your Git history and make your PR unuseable.
Use "git rebase" instead. See this forum post for further details:
https://community.openhab.org/t/rebase-your-code-or-how-to-fix-your-git-history-before-requesting-a-pull/129358

Add one or more appropriate labels to make your PR show up in the release notes.
E.g. enhancement, bug, documentation, new binding
This can only be done by yourself if you already contributed to this repo.

If your PR's code is not backward compatible with previous releases (which
should be avoided), add a message to the release notes by filing another PR:
https://github.com/openhab/openhab-distro/blob/main/distributions/openhab/src/main/resources/bin/update.lst

# Title
-[liquidcheck] Initial contribution

# Description

New Binding to connect the Liquid Check device from SI-Electronics GmbH with openHAB.
Then sensor will provide data from tanks like content and level.
Furthermore it will provide data about its own status like  IP, hostname, firm and hardware version,...

# Testing
https://community.openhab.org/c/add-ons ](https://github.com/marcelGoerentz/openhab-addons/releases/tag/V1.0.0)

### CODE REVIEW

File: bundles/pom.xml [hunk 1]
```diff
@@ -5,643 +5,1268 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
+
```
Comment (digitaldan) [inline bundles/pom.xml:8]: So it looks like some formatter on your side added a bunch of empty lines to this file, you probably want to revert this one file and try again to just add the one line you need.

File: bom/openhab-addons/pom.xml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-
```
Comment (digitaldan) [inline bom/openhab-addons/pom.xml:4]: Please add back in this line so the only change is the addition of your binding

**Comment by digitaldan** [comment]
Hi @marcelGoerentz , this PR looks much better :-)   I left a couple general formatting/git comments only, as a rule of thumb when changing files outside your binding that affect all other bindings (like the addons pom) you only want to change whats needed to add your binding and nothing else, this helps avoid merge conflicts for others and lessens the chance of other things breaking.

**Comment by marcelGoerentz** [comment]
Hi @digitaldan,
Thank you for your comments.
I will fix those violations.
So will I have to open a new PR ?
The added line in the .gitattribute file would help all windows users as they have to add it by themselves. 😉

Thanks for your patience and help.
Much appreciate it.

**Comment by digitaldan** [comment]
> So will I have to open a new PR ?

No, just update your branch, when you push it to your github repo, this PR automatically gets updated (since its based on your live branch).

>The added line in the .gitattribute file would help all windows users as they have to add it by themselves. 

Yeah, i'll defer to one of our main reviewers on that, i'm not sure if there are any other consequences and why it was not there before.

**Comment by digitaldan** [comment]
Also, don't worry about being new to git/github and asking questions, we all have to start somewhere and it gets easier after the first successful PR :-)

**Comment by jlaur** [comment]
> > The added line in the .gitattribute file would help all windows users as they have to add it by themselves.
> 
> Yeah, i'll defer to one of our main reviewers on that, i'm not sure if there are any other consequences and why it was not there before.

Please see https://github.com/openhab/openhab-addons/pull/11999#discussion_r901868526

**Comment by marcelGoerentz** [comment]
Hey guys,

I've rebased my branch and fixed the violations.
I hope, that I didn't miss anything :)

**Comment by marcelGoerentz** [comment]
Sorry for those commits.
These editors are driving me nuts with their auto filling...

File: .gitattributes [hunk 3]
```diff
@@ -1,2 +1,3 @@
 *.java text=auto
```
[the remark this answers is not in the document]
Comment (marcelGoerentz) [reply .gitattributes]: I've reverted the file

File: bom/openhab-addons/pom.xml (hunk 2 above)
Comment (marcelGoerentz) [reply bom/openhab-addons/pom.xml]: File reverted to original format

File: bundles/pom.xml (hunk 1 above)
Comment (marcelGoerentz) [reply bundles/pom.xml]: File reverted to original format

**Comment by marcelGoerentz** [comment]
Hey guys, I guess this is ready for review.
I'm waiting for your comments 😁.

**Pull request closed without merging**